### ISSUE TITLE
Is it okay if I send you pull request with formatted code?

### ISSUE THREAD

**Issue by SachiraChin** [julielerman/dotnetconfms2020#1]
Please do not misunderstand my intent, since this code is targeted for C# developers, I thought it'd be nice if its formatted in the way its usually used in C# projects. I understand if you want to keep it as is. Thank you!

**Comment by julielerman** [comment]
I appreciate that you asked and were a little concerned I might be offended. I did use codemaid to format most of it and it looks the way I'm used to seeing it. I'm guessing its the javascript-y open braces not on their own line, maybe? Anyway if it is going to make your eyes pop out to look at the code as it is and you think it would be a good service to future readers of my code and you have the time, sure.

**Comment by SachiraChin** [comment]
@julielerman I think I didn't think far as it's anything bad. I thought there may be reason which it was written that way which shouldn't be changed, for example for specific audience. That's why I put it that way. 😊 Please be assured that I didn't think you would argue about it in any nonsensical way. 

I will send you a pull request later today, you can decide if its worth merging. I will try to update the markdown with details of your session once video is available to refer.

**Comment by julielerman** [comment]
I do plan to update the readme when the video is available. Thanks for having my back! ;)

**Comment by SachiraChin** [comment]
@julielerman You are welcome but this is nothing compared to effort you put to it. 😊 I'm just doing very tiny, small part in my spare time.

**Comment by julielerman** [comment]
and that's how GitHub works! :)

**Comment by SachiraChin** [comment]
Agreed. 😁

**Comment by julielerman** [comment]
because of your comment about moving code around, and I didn't want to have to figure out what else had changed besides returns and spacing in all of the files, I took 2 minutes to whip through the files and let code maid fix up the formatting. I have just pushed that change and therefore won't take the PR. I appreciate that you pointed out that I could have made that better and that you did the PR to fix it.
